scripts/issue-pr.ps1

```
@@ -0,0 +1,72 @@
+- [ ] `pnpm run check`
+- [ ] `cargo test -p pomodoroom-core`
+- [ ] `cargo test -p pomodoroom-cli -- --test-threads=1`
+- [ ] Manual test done
+
+## Screenshots / Logs (if needed)
+
+## Risks
+- 
+"@
+
+$args = @("pr", "create", "--base", $Base, "--title", $title, "--body", $body)
```

PSScriptAnalyzer (1.23.0)

[warning] The Variable 'args' is an automatic variable that is built into PowerShell, assigning to it might have undesired side effects. If assignment is not by design, please use a different name.

(PSAvoidAssignmentToAutomaticVariable)